test/fixtures/templates/public/rds-cluster.yaml

```diff
@@ -0,0 +1,35 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Description: MU database in a specific environment
+Parameters:
+  DatabaseMasterUsername:
+    Type: String
+    NoEcho: true
+    Description: Username of database
+  DatabaseName:
+    Type: String
+    Description: Name of database
+  DatabaseMasterPassword:
+    Type: String
+    NoEcho: true
+    Description: Password of database
+
+Resources:
+  DBCluster:
+    Type: AWS::RDS::DBCluster
+    Condition: IsClustered
+    Properties:
+      MasterUsername: !Ref DatabaseMasterUsername
+      MasterUserPassword: !Ref DatabaseMasterPassword
+      DatabaseName: !Ref DatabaseName
+      Engine: aurora
+      EngineMode: serverless
+      ScalingConfiguration: 
+        AutoPause: true
+        MaxCapacity: 8
+        MinCapacity: 2
+#        SecondsBeforeAutoPause: 300
```

Comment:
Can we remove this commented line? Just to keep it clean?

Reply:
I removed the whole section, that's there because of a spec bug (#365) which was how I originally stumbled upon this problem.  I also noticed we're not catching the undefined Conditional, I'll remove that line and create another issue for that.